templates/Pages/contact.php

```diff
@@ -0,0 +1,76 @@
+<?php $this->assign('title', 'Contact'); ?>
+
+<div class="landing landing-contact">
+    <h1><?php echo __('Contact') ?></h1>
+
+    <p>
+        You can use any of the following channels to stay up to date with IO500 and to discuss with the community.
+    </p>
+
+    <ul>
+        <li>
+            <?php
+            echo $this->Html->link(__('Mailing List'),
+                'http://lists.io500.org/listinfo.cgi/io500-io500.org',
+                [
+                    'class' => 'button-highlight',
+                    'target' => '_blank'
+                ]
+            );
+            ?>
+        </li>
+        <li>
+            <?php
+            echo $this->Html->link(__('Mail Committee'),
```

Comment:
I disagree that a "button" should be used to email the committee it suggests being a contact form of some sort and "Mail" instead of a symbol is suboptimal.
Better to list the email and it being a link.

Reply:
Julian, can you please clarify your objection about "Mail committee" being a link?  The "mailto:" URL has existed for decades, and allows users to use their own email client to write the email, rather than having a web form (which is worse because it forces the user to use the limited functionality in the form, and potentially exposes the system as a spam relay).

I agree it makes sense for the various addresses are also listed in plain text below or beside the buttons:

    [Mailing list]    io500@lists.io500.org  (subscribe)
    [Slack channel]   io500workspace.slack.com (subscribe)
    [Mail committee]  committee@io500.org
    [Twitter]         @io500benchmark

Reply:
That said, I would be OK to land these changes as-is, so that the "Contact" page is at least available, and can be improved in a separate patch.